app/inputs/multi_value_input.rb

```diff
@@ -0,0 +1,64 @@
+class MultiValueInput < SimpleForm::Inputs::Base
```

Comment:
Which is to say, once we're happy with this stuff I want it pushed to our local hydra editor branch and then we can properly test in both places where it makes sense.